src/export/bops/schema/utils.ts

```diff
@@ -0,0 +1,140 @@
+import { z } from "zod";
+
+import {
+  ComponentType,
+  EnrichedCrumb,
+  Node,
+  QuestionAndResponses,
+  Response,
+} from "../../../types";
+
+export const isSchemaType = (type: ComponentType) =>
+  [ComponentType.Page, ComponentType.List].includes(type);
+
+/**
+ * Parse the data of a node which uses a schema for responses
+ */
+const parseNodeData = (node: Node) => {
```

Comment:
This could maybe be a bit easier if we had the `Schema` types from the frontend here, but this is a nice way of parsing an `unknown` into the type we expect.

In an ideal world, the Flow and Breadcrumb types would be shared and node-specific, e.g. `EncrichedCrumb<Page>` or `NodeData<List>`.